code/modules/mob/living/carbon/human/say.dm

```diff
@@ -133,7 +133,7 @@
 	if(silent || (sdisabilities & MUTE))
 		message = ""
 		speech_problem_flag = 1
-	else if(!src.is_diona() && istype(wear_mask, /obj/item/clothing/mask))
+	else if(!src.is_diona() && (istype(wear_mask, /obj/item/clothing/mask) || is_muzzle(wear_mask)))

```

Comment:
_Noooo._ This is dangerous and will most likely runtime. This right here breaks what you're trying to do. DM does not have multiple inheritance, and this chain of execution very specifically expects an object of type mask. Your definitions of the two variables above won't help at all.

One way to fix this would be to use safe defaults here:
```dm
	else if (!is_diona() && is_muzzle(wear_mask))
		if (istype(wear_mask, /obj/item/clothing/mask))
			var/obj/item/clothing/mask/M = wear_mask
			if (M.voicechange)
				message = pick(M.say_messages)
				verb = pick(M.say_verbs)
				speech_problem_flag = 1
		else
			message = pick(list("Mmfph!", "Mmmf mrrfff!", "Mmmf mnnf!"))
			verb = pick(list("mumbles", "says"))
			speech_problem_flag = 1
```